### PR TITLE
[WIP] swisscom api

### PR DESCRIPTION
**Pull request by lgavillet**

### CODE REVIEW

File: lib/mote_sms/transports/swisscom_transport.rb [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    # options - The Hash with additional, per delivery options.
+    #
+    # Returns Array with params.
+    def post_params(message, options)
+      {
+        outboundSMSMessageRequest: {
+          senderAddress: "tel:#{message.from.to_number}",
+          address: prepare_numbers(message.to),
+          outboundSMSTextMessage: { message: message.body },
+          clientCorrelator: options[:messageid]
```
Comment (lwe) [inline lib/mote_sms/transports/swisscom_transport.rb]: @lgavillet should we maybe add a default messageid if none is used?

Comment (lgavillet) [inline lib/mote_sms/transports/swisscom_transport.rb]: dunno if one is required or not. Worst case, it's removed by the compact the line after

File: lib/mote_sms/transports/swisscom_transport.rb [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    # `allow_adaption: true` as default.
+    #
+    # Examples:
+    #
+    #    MoteSMS::MobileTechnicsTransports.defaults[:messageid] = ->(msg) { "#{msg.from}-#{SecureRandom.hex}" }
+    #
+    # Returns Hash with options.
+    def self.defaults
+      @@options ||= {
+        allow_adaption: true,
```
Comment (lwe) [inline lib/mote_sms/transports/swisscom_transport.rb]: allow_adaption is something for MobileTechnicsTransport only

File: lib/mote_sms/transports/swisscom_transport.rb [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    # Readable attributes
+    attr_reader :endpoint, :api_key, :from_number, :options
+
+    # Public: Global default parameters for sending messages, Procs/lambdas
+    # are evaluated on #deliver. Ensure to use only symbols as keys. Contains
+    # `allow_adaption: true` as default.
+    #
+    # Examples:
+    #
+    #    MoteSMS::MobileTechnicsTransports.defaults[:messageid] = ->(msg) { "#{msg.from}-#{SecureRandom.hex}" }
```
Comment (lwe) [inline lib/mote_sms/transports/swisscom_transport.rb]: ^^ comment

**Pull request merged**